goat/backend/eval.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+from pathlib import Path
+
+from fastchat.llm_judge.gen_model_answer import run_eval
+from fastchat.utils import str_to_torch_dtype
+from lm_eval import evaluator
+from lm_eval.models.huggingface import HFLM
+
+from goat.backend.add_results import add_results
+from goat.utils.database_helper import DatabaseHelper
+
+
+def eval(model_name: str, precision: str, generate_fastchat: bool) -> None:
+    lm = HFLM(pretrained=model_name, dtype=precision)
+    taskname = "goat"
+    results = evaluator.simple_evaluate(model=lm, tasks=[taskname])
+
+    model_id = model_name.replace("/", "__")
+    Path(f"goat/backend/results/{model_id}").mkdir(exist_ok=True)
+    lm_eval_output_file = f"goat/backend/results/{model_id + '_lm_eval'}.json"
```

Comment:
Why did you create folder with `model_id`, but store results in `model_id_lm_eval.json`. Probably, this json should be inside model directory